src/js/ia/IADevPipeline.js

```diff
@@ -144,6 +144,26 @@
                 }
             });
 
+            $("body").on("click", "#beta_install", function(evt) {
+                var prs = [];
+                $(".dev_pipeline-column__list .selected").each(function(idx) {
+                    var temp_pr = $.trim($(this).find(".item-activity a").attr("href"));
+                    var temp_pr_id = temp_pr.replace(/.*\//, "");
+                    var temp_repo = temp_pr.replace(/.*zeroclickinfo\-/, "").replace(/\/[a-zA-Z0-9]*\/[a-zA-Z0-9]*\/?/, "");
```

Comment:
can this be the full repo name?  That would make the beta server stuff easier to do.